src/parser/parser.cpp

```diff
@@ -211,7 +214,7 @@ void Parser::ParseQuery(const string &query) {
 						auto result = ext.parse_function(ext.parser_info.get(), query_statement);
 						if (result.type == ParserExtensionResultType::PARSE_SUCCESSFUL) {
 							auto statement = make_uniq<ExtensionStatement>(ext, std::move(result.parse_data));
-							statement->stmt_length = query_statement.size();
+							statement->stmt_length = query_statement.size() - 1;
 							statement->stmt_location = 0;
```

Comment:
This also seems incorrect to me in case of multiple statements - `stmt_location` should be the byte position where the statement starts

Reply:
Okay added this in. I was surprised it worked.. I guess it's not used in the actual parsing but only reporting error purposes?

Reply:
Yes it is only be used for error reporting currently